tasties_app/tests/test_Comment.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from tasties_app.models import Comment, Recipe, Category
+from django.contrib.auth.models import User
+from django.utils import timezone
+from datetime import date
+
+
+@pytest.fixture
+def generate_data():
+    """This fixture create the data we need for tests
+    and save it in the data base. """
+    user3 = User.objects.create_user(username='user3', password='password')
+    test_data = ['Test Recipe1', user3, 'Test Description1', 'Test Directions1', 1, 'recipe.jpeg']
+    category1 = Category.objects.create(category_name="1")
```

Comment:
We can drop this and have a recipe without a category. just to make this fixture smaller.

Reply:
@WaseemAbuSobe you missed this comment.